### DISCUSSION TITLE
Request to joining Team

### DISCUSSION THREAD

**Discussion by gioaprilino** [ROM-builders/temporary#22423] [General]
My successful builds
[cirrus-ci.com/build/5769570170961920](https://cirrus-ci.com/build/5769570170961920)
[cirrus-ci.com/build/5761906774900736](https://cirrus-ci.com/build/5761906774900736)

Source code to build my ROM
[DT](https://github.com/giosaja96/device_xiaomi_mojito/tree/13)
[VT](https://github.com/giosaja96/vendor_xiaomi_mojito/tree/13)
[kernel](https://github.com/fiqri19102002/android_kernel_xiaomi_mojito)

My github username
[@giosaja96](https://github.com/giosaja96)
Local manifest
[giosaja96/local_manifest](https://github.com/giosaja96/local_manifest)
Telegram username [@gioaprilino](https://t.me/gioaprilino)

**Comment by Apon77** [comment]
Please follow #learn instruction from telegram group.
